libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/snapshotter/model/MapboxSnapshotterOptions.kt

```diff
@@ -132,6 +141,14 @@ class MapboxSnapshotterOptions private constructor(
         fun styleUri(styleUri: String): Builder =
             apply { this.styleUri = styleUri }
 
+        /**
+         * apply routePrecision to the builder
+         * @param routePrecision String
+         * @return Builder
+         */
+        fun routePrecision(routePrecision: Int): Builder =
```

Comment:
Same as noted above. Should we check the `precision` range and throw an `IllegalArgumentException` if not `5` or `6`? 🤔

We should also be adding the `@IntRange(from = 5, to = 6)` annotation.

Reply:
Will add `IntRange` and throw an exception if some other value is passed.